gems/aws-sdk-s3/1/manifest.yaml

```diff
@@ -0,0 +1,2 @@
+dependencies:
+  - name: aws-sdk-core
```

Comment:
I think the `manifest.yaml`s are not necessary.

`aws-sdk-s3` and `-sqs` declare the dependency to aws-sdk-core in the gemspec. ref: https://rubygems.org/gems/aws-sdk-sqs
`manifest.yaml` is a file to declare the dependencies that don't appear in the gemspec, such as standard libraries, so `aws-sdk-core` is not necessary to put in this place in this case.

Reply:
Thanks 🎍
I understand about manifest.yaml.
I'll fix it later.